spec/factories/users.rb

```diff
@@ -31,6 +33,8 @@
 #
 FactoryBot.define do
   factory :user do
+    first_name { Faker::Internet.name.first_name }
```

Comment:
This should be the correct format: https://github.com/faker-ruby/faker/blob/main/doc/default/name.md